client/overview/task-list/tasks/update-business-details-task.tsx

```diff
@@ -26,6 +27,9 @@ export const getUpdateBusinessDetailsTask = (
 	const accountDetailsPastDue = 'restricted' === status && pastDue;
 	const hasMultipleErrors = 1 < errorMessages.length;
 	const hasSingleError = 1 === errorMessages.length;
+	const accountLinkWithSource = addQueryArgs( accountLink, {
+		source: 'update-business-details',
```

Comment:
Could we be more specific about the source? Maybe something like `overview-page__update-business-details-task`

Reply:
Addressed in https://github.com/Automattic/woocommerce-payments/pull/8668/commits/6805615ab089eb023e57737d80ae7d96248ed2c4